.config/spellcheck.toml

```diff
@@ -7,7 +7,8 @@ search_dirs = ["."]
 skip_os_lookups = true
 use_builtin = true
 tokenization_splitchars = "\",;:.!?#(){}[]|/_-‒'`&@§¶…"
-extra_dictionaries = ["forest.dic", "en_US.dic"]
+# ignore.dic stores unrecogonized words in generated files (e.g. protobuf)
```

Comment:
Any better solutions? Could not find an option to ignore certain files

Reply:
Hmm, yeah, I don't see a better solution.